cql/ManageCommonAbnormality.cql

```diff
@@ -22,6 +22,10 @@ include CollateManagementData version '1.1.0' called Collate
 include AutogeneratedTableLookup version '1.0.0' called RiskTableLookup
 include CCSMCommonFunctions version '1.0.0' called CCF
 
+codesystem "SNOMED-CT": 'http://snomed.info/sct'
```

Comment:
It looks like all the changes in this file were made for debugging. Should they be removed?

Reply:
@mrnosal there are still changes to this file that do not look like they are intended to be merged. Looking at a diff of this PR, please see the extra line added at the top, and the `debugCytologyInterpretedAsAscus` and `debugCytologyAloneInterpretedAsAscus` expressions.

Reply:
Outstanding comment - it seems unnecessary to touch this file within this PR.

Reply:
I've removed these.

Reply:
No. The extra line at the top _was not removed_. Please see https://github.com/ccsm-cds-tools/ccsm-cds-with-tests/pull/134/files#diff-b2088cb233e1f2b7f6171c186140d966d450ff6873e21718e3839d4c9346343b. ManageCommonAbnormality.cql is changed by this PR. Why?

Reply:
Addressed in https://github.com/ccsm-cds-tools/ccsm-cds-with-tests/pull/134/commits/92c7d5ab722eddc485e04c8499a6aa7990178f99.